src/plugins/intel_gpu/tests/fusions/fusion_validity_test.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils.h"
+#include "fusion_test_common.hpp"
+
+#include <intel_gpu/primitives/input_layout.hpp>
+#include <intel_gpu/primitives/quantize.hpp>
+#include <intel_gpu/primitives/eltwise.hpp>
+#include <intel_gpu/primitives/data.hpp>
+#include <intel_gpu/primitives/resample.hpp>
+
+#include <cmath>
+
+using namespace cldnn;
+using namespace ::tests;
+
+namespace {
+struct fusing_test_params {
+    tensor in_shape;
+    tensor out_shape;
+    data_types data_type;
+    format input_format;
+    format output_format;
+    resample_type type;
+    data_types default_type;
+    format default_format;
+    size_t expected_fused_primitives;
+    size_t expected_not_fused_primitives;
+};
+
+#define CASE_RESAMPLE_FP16_1 { 1, 16, 64, 64 }, { 1, 16, 128, 128 }, data_types::f16, format::b_fs_yx_fsv16, format::bfyx, resample_type::nearest, data_types::f16, format::bfyx
+#define CASE_RESAMPLE_FP16_2 { 1, 2,  32, 32 }, { 1, 2,   64,  64 }, data_types::f16, format::b_fs_yx_fsv16, format::bfyx, resample_type::nearest, data_types::f16, format::bfyx
+#define CASE_RESAMPLE_FP32_1 { 1, 16, 32, 32 }, { 1, 16,  64,  64 }, data_types::i8,  format::b_fs_yx_fsv32, format::bfyx, resample_type::nearest, data_types::i8, format::bfyx
+#define CASE_RESAMPLE_FP32_2 { 1, 2,  32, 32 }, { 1, 2,   64,  64 }, data_types::i8,  format::b_fs_yx_fsv32, format::bfyx, resample_type::nearest, data_types::i8, format::bfyx
+
```

Comment:
Could you add some more description on this test? Its main purpose is to validate whether LT_ALIGNED_READ is replaced with LT_UNALIGNED when necessary. But it is not mentioned.

Reply:
Added